monkey/monkey_island/cc/resources/configuration_import.py

```diff
@@ -39,6 +39,8 @@ def form_response(self):
 
 
 class ConfigurationImport(flask_restful.Resource):
+    # API Spec: This is an action and there's no "resource"; RPC-style endpoint?
```

Comment:
This should probably be merged with `monkey/monkey_island/cc/resources/island_configuration.py`